docs/edgeql/ddl/annotations.rst

```diff
@@ -41,8 +41,9 @@ has an annotation defined on it, the descendants of that schema item will
 not automatically inherit the annotation.  Normal inheritance behavior can
 be turned on by declaring the annotation with the *INHERITABLE* qualifier.
 
-The following subcommands are allowed in the
-``CREATE ABSTRACT ANNOTATION`` block:
+The core of the command is identical to the :ref:`SDL annotation
```

Comment:
```suggestion
The core of the command is identical to the :ref:`SDL annotation
```

"The core of the command" reads weird. Maybe replace with something along "Most sub-commands and options of this command ..."?

Reply:
Wording changed